templates/component.tsx.ejs

```diff
@@ -17,17 +32,11 @@ import { <%= props.pascalName %>Props } from "./<%= props.name %>.props"
  */
 export function <%= props.pascalName %>(props: <%= props.pascalName %>Props) {
   // grab the props
-  const { preset = "primary", tx, text, style: styleOverride, ...rest } = props
-
-  // assemble the style
-  const viewPresetToUse = viewPresets[preset] || viewPresets.primary
-  const textPresetToUse = textPresets[preset] || textPresets.primary
-
-  const viewStyle = { ...viewPresetToUse, ...styleOverride }
-  const textStyle = textPresetToUse
+  const { tx, text, style, ...rest } = props
+  const textStyle = { }
 
   return (
-    <TouchableOpacity style={viewStyle} {...rest}>
+    <TouchableOpacity style={style} {...rest}>
```

Comment:
My inclination is to make this a straight up `<View>` and lose the `extends TouchableOpacityProperties` in the props above, but my GAFO is relatively low if others want to keep it.